src/spikeinterface/extractors/mdaextractors.py

```diff
@@ -180,6 +180,13 @@ def get_traces(
 class MdaSortingExtractor(BaseSorting):
     """Load MDA format data as a sorting extractor.
 
+    NOTE: As in the MDA format, the max_channel property indexes the channels that are given as input
+    to the sorter.
+    If sorting was run on a subset of channels of the recording, then the max_channel values are
+    based on that subset, so care must be taken when associating these values with a recording.
+    If additional sorting segments are added to this sorting extractor after initialization,
+    then max_channel will not be updated. The max_channel indices begin at 1.
```

Comment:
Should we then set the property with a -1?

Reply:
Probably. Though if you are familiar with the MDA format, you might know and expect that the channel indices begin at 1. I leave that for you to decide

Reply:
No strong preference here! Just thinking that if downstream or during NWB export we assume these are channel indices we would assume they start from 0.

@magland any thought here?

Reply:
Letàs leave 1-based, since this is a reader :)